plugins/katello/3.16/release_notes/release_notes.md

```diff
@@ -25,6 +25,10 @@ For the full release notes, see the [Changelog](https://github.com/Katello/katel
     * MongoDB will be removed in Katello 4.0
     * The Foreman installer MongoDB Storage Engine Migration Hook will be removed for Katello 4.0 
 
+## Notes
+
+* Katello 3.16 is not compatible with EL8, but Katello 4.0 will be.
```

Comment:
This might be confusing since it can manage EL8 clients. I'd explicitly state it's about deploying Katello itself.